### PR TITLE
docs(screenreaderonly): improve ScreenReaderOnly documentation page

### PR DESCRIPTION
**Pull request by massao**

### CODE REVIEW

File: packages/core/src/ScreenReaderOnly/README.mdx [hunk 1]
```diff
@@ -18,56 +16,26 @@ This is a helper component to add content and components which is only visible f
 
 ### Import
```
Comment (bgutsol) [inline packages/core/src/ScreenReaderOnly/README.mdx:17]:
```suggestion
## Import
```

**Review by bgutsol — COMMENTED** [review]
A couple of comments:

I'd suggest changing
`## How to use ScreenReaderOnly` to third level title `### How to use ScreenReaderOnly`

**Comment by gui-santos** [comment]
Looks good, I just +1 @bgutsol  suggestions

**Review by gui-santos — APPROVED** [review]
Hahaha you submitted the suggestions at the same time that I made my comment

**Review by bgutsol — APPROVED** [review]

**Pull request merged**